runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/variables.actions.ts

```diff
@@ -122,6 +123,50 @@ const getVariablesActions: () => Record<string, IBpmnAction> = () => ({
             },
         },
     },
+    'variables.assignMultipleGlobalVariable': {
```

Comment:
it should not be another action but single action that handles one or more variables